src/qwerty.py

```diff
@@ -8,4 +8,11 @@
 # Assume that Jenny's intended message only contains letters

```

Comment:
Solution works for both letters and other characters, not necessary but probably good.